annotation/tests/test_distribution.py

```diff
@@ -1277,3 +1283,191 @@ def test_get_page_number_combinations(
     expected: List[int],
 ) -> None:
     assert get_page_number_combinations(file_pages, user_pages) == expected
+
+
+# ADDITIONAL TESTS
+
+USERS = [
+    {
+        "user_id": "405ef0e2-b53e-4c18-bf08-c0871615d99d",
+        "default_load": 100,
+        "overall_load": 0,
+    },
+    {
+        "user_id": "ba3e0ccf-8661-4ed3-892b-7c291160f631",
+        "default_load": 96,
+        "overall_load": 0,
+    },
+    {
+        "user_id": "6e2ce9ac-2fe8-4cc7-bdcd-bac90faa9247",
+        "default_load": 95,
+        "overall_load": 0,
+    },
+    {
+        "user_id": "8c8a333e-d19a-492a-9e78-5df4bec0ec8b",
+        "default_load": 92,
+        "overall_load": 0,
+    },
+    {
+        "user_id": "ba38c547-f7f2-4ece-b56f-9926e34b159a",
+        "default_load": 90,
+        "overall_load": 0,
+    },
+    {
+        "user_id": "c080408e-0077-4c20-b520-bd4b1541ca56",
+        "default_load": 100,
+        "overall_load": 50,
+    },
+    {
+        "user_id": "c080408e-0077-4c20-b520-bd4b1541ca57",
+        "default_load": 100,
+        "overall_load": 100,
+    },
+    {
+        "user_id": "c080408e-0077-4c20-b520-bd4b1541ca58",
+        "default_load": 0,
+        "overall_load": 100,
+    },
+    {
+        "user_id": "c080408e-0077-4c20-b520-bd4b1541ca59",
+        "default_load": 0,
+        "overall_load": 50,
+    },
+    {
+        "user_id": "c080408e-0077-4c20-b520-bd4b1541ca60",
+        "default_load": 100,
+        "overall_load": 100,
+    },
+]
+
+ANNOTATION_TASKS_CHOOSE_VALIDATORS = [
+    {
+        "deadline": None,
+        "file_id": 4,
+        "is_validation": False,
+        "job_id": 1,
+        "pages": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15],
+        "user_id": "405ef0e2-b53e-4c18-bf08-c0871615d99d",
+        "status": TASKS_STATUS,
+    },
+    {
+        "deadline": None,
+        "file_id": 5,
+        "is_validation": False,
+        "job_id": 1,
+        "pages": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+        "user_id": "c080408e-0077-4c20-b520-bd4b1541ca60",
+        "status": TASKS_STATUS,
+    },
+    {
+        "deadline": None,
+        "file_id": 3,
+        "is_validation": False,
+        "job_id": 1,
+        "pages": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16],
+        "user_id": "c080408e-0077-4c20-b520-bd4b1541ca58",
+        "status": TASKS_STATUS,
+    },
+]
+
+EXPECTED_OUTPUT = [
+    [
+        {
+            "user_id": "ba3e0ccf-8661-4ed3-892b-7c291160f631",
+            "default_load": 96,
+            "overall_load": 0,
+        },
+        {
+            "user_id": "c080408e-0077-4c20-b520-bd4b1541ca59",
+            "default_load": 0,
+            "overall_load": 50,
+        },
+    ],
+    [
+        {
+            "user_id": "6e2ce9ac-2fe8-4cc7-bdcd-bac90faa9247",
+            "default_load": 95,
+            "overall_load": 0,
```

Comment:
Expected output is always related to a certain test, no need to keep in in a module level. Additionally, all data you prepared above should be used in test directly if you dont utilize in multiple places